app/nodes/controller.js

```diff
@@ -106,14 +106,13 @@ export default Controller.extend({
       return node;
     });
 
-    if (!filterQuery) {
-      return allNodes;
+    let result = allNodes;
+    if (filterQuery) {
+      result = allNodes.filter((node) => {
+        return node.address.match(filterQuery) || node.userAgent.match(filterQuery);
```

Comment:
wouldn't this return a non sorted subset of nodes?

Reply:
Ordering is indeed done here: https://github.com/BitcoinUnlimited/cashnodes-frontend/pull/12/files#diff-c2c9bda95567f5f813457e79644e6b0fR115

Reply:
you are right indeed